plugin/simpl.vim

```diff
@@ -81,3 +81,17 @@ call simpl#register(
       \ 'scheme',
       \ { s -> printf("(load \"%s\")\n", s)},
       \ {-> '(load) '})
+
+function s:fsharp_load(file) abort
+  let l:module_name = fnamemodify(a:file, ':r')
+  if l:module_name =~# '^\u'
```

Comment:
Can you explain this test for a module name starting with an upper-case character and why it leads to `#load` + `open`?

Reply:
That was me misunderstanding the F# interactive documentation, I've removed the if since it wasn't needed. But the `open` is because when F# interactive loads a file, it puts everything in a module with the same name as the file, so to run your script you have to type `ModuleName.excitingCalculation`, or first do a `open`, adding an `open` automatically saves on typing.